mesh/token_bucket.c

```diff
@@ -0,0 +1,56 @@
+#include "token_bucket.h"
+
+#include <ell/ell.h>
+
+
+static const uint32_t time_quantum_ms = 1;
```

Comment:
Don't user globals, create a struct for that and allow multiple instances of the filter.